src/components/MyKiva/GoalCard.vue

```diff
@@ -5,68 +5,74 @@
 			'goal-card-bg': !userHasGoal,
 		}"
 	>
-		<div :class="{'tw-mx-auto tw-relative achievement-card-bg': userHasGoal}">
-			<span
-				:class="{'tw-absolute tw-top-1 tw-left-1': userHasGoal}"
-				class="
-					tw-inline-flex tw-items-center tw-gap-1
-					tw-mb-2 tw-rounded-md
-					tw-bg-eco-green-1 tw-px-1.5 tw-py-1"
-				title="Your lending reach"
-			>
-				<KvMaterialIcon
-					class="tw-w-2 tw-h-2 tw-shrink-0"
-					:icon="mdiCheckCircleOutline"
-				/>
-				<span class="tw-text-primary tw-font-medium tw-align-middle" style="font-size: 0.875rem;">
-					My goal
+		<template v-if="isGoalComplete">
+			<!-- Change with goal complete component from MP-2014 -->
+			<p>Goal Complete Component</p>
```

Comment:
Given conversation in other thread, will we actually ever show the completed goal card on MyKiva? Please look at tickets.